### DISCUSSION TITLE
PRs for new features vs keeping it simple

### DISCUSSION THREAD

**Discussion by jyegerlehner** [karpathy/build-nanogpt#34] [General]
Would PRs for new features (e.g. resuming training) be welcome? Or is the intention for this repo to keep it as simple as possible and avoid code bloat?

I find myself preferring this codebase over other frameworks (e.g. HF transformers) for some experiments because it is more easily understood, but of course need to add a few things. But if the doors are thrown open to changes it can quickly become complicated. So I could see going either way.

**Comment by karpathy** [comment]
It's the intention of this repo to not really change. I'll merge any bug fixes, clarifications, or things like that.
